src/redux/actions/platformActions.js

```diff
@@ -49,65 +52,257 @@ const authorizeUser = appName => dispatch =>
+              context: ['export', 'completed', 'descriptionGlobal'],
+              count: existingExports.length
+            }),
+            dismissable: true
+          }
+        }
+      }
     });
 
 /**
- * Get a specific, or all, export status.
+ * Remove all existing exports. Includes toast notifications.
  *
- * @param {object} options Apply polling options
+ * @param {Array<{ id: string }>} existingExports
  * @returns {Function}
  */
-const getExportStatus =
+const removeExistingExports = existingExports => dispatch =>
+  dispatch({
+    type: 'DELETE_EXPORT',
+    payload: Promise.all(existingExports.map(({ id }) => platformServices.deleteExport(id))),
+    meta: {
+      notifications: {
+        rejected: {
+          variant: 'warning',
+          title: translate('curiosity-toolbar.notifications', {
+            context: ['export', 'error', 'title']
+          }),
+          description: translate('curiosity-toolbar.notifications', {
+            context: ['export', 'error', 'description']
+          }),
+          dismissable: true
+        }
+      }
+    }
+  });
+
+/**
+ * Get a status from any existing exports. Display a confirmation for downloading, or ignoring, the exports.
+ * Includes toast notifications.
+ *
+ * @param {object} options
+ * @returns {Function}
+ */
+const getExistingExportsStatus =
   (options = {}) =>
-  dispatch =>
-    dispatch({
+  dispatch => {
+    const onYes = allResults => {
+      dispatch(removeNotification(`swatch-exports-status`));
+      getExistingExports(allResults)(dispatch);
+    };
+    const onNo = allResults => {
+      dispatch(removeNotification(`swatch-exports-status`));
+      removeExistingExports(allResults)(dispatch);
+    };
+
+    return dispatch({
       type: platformTypes.SET_PLATFORM_EXPORT_STATUS,
-      payload: platformServices.getExportStatus(undefined, undefined, {
-        ...options,
-        poll: { ...options.poll, status: setExportStatus(dispatch) }
-      })
+      payload: platformServices.getExistingExportsStatus(undefined, options).then(successResponse => {
+        const isCompleted =
+          !successResponse?.data?.data?.isAnythingPending && successResponse?.data?.data?.isAnythingCompleted;
+        const isPending =
+          successResponse?.data?.data?.isAnythingPending && !successResponse?.data?.data?.isAnythingCompleted;
+        const isAnythingAvailable = isCompleted || isPending || false;
+
+        const completed = successResponse?.data?.data?.completed || [];
+        const pending = successResponse?.data?.data?.pending || [];
+        const totalResults = completed.length + pending.length;
+
+        if (isAnythingAvailable && totalResults) {
+          dispatch(
+            addNotification({
+              id: `swatch-exports-status`,
+              title: `${totalResults} existing reports are available`,
+              description: (
+                <div aria-live="polite">
```

Comment:
required: 
- since this involves components review moving the alert format into the toolbarFieldExport file
- locale strings needed